sht4x/sht4x.go

```diff
@@ -0,0 +1,79 @@
+// Package sht4x provides a driver for the SHT4x digital humidity sensor series by Sensirion.
+// Datasheet: https://www.sensirion.com/media/documents/33FD6951/64D3B030/Sensirion_Datasheet_SHT4x.pdf
+package sht4x
+
+import (
+	"time"
+
+	"tinygo.org/x/drivers"
+)
+
+const DefaultAddress = 0x44
+
+const (
+	// single-shot, high-repeatability measurement
+	commandMeasurement = 0xfd
+)
+
+// Device represents a SHT4x sensor
+type Device struct {
+	bus     drivers.I2C
+	address uint8
+}
+
+// New creates a new SHT4x connection. The I2C bus must already be
+// configured.
+func New(bus drivers.I2C, addr uint8) Device {
```

Comment:
I would be better to follow the pattern used by most drivers here, which is 

```go
type Device struct {
	bus     drivers.I2C
	Address uint8
}

func New(bus drivers.I2C) Device {
return Device{
		bus:     bus,
		Address: DefaultAddress,
	}
}
```

If the dev wants to set the address to something else, it is then a matter of

```go
d := sht4x.New(machine.I2C0)
d.Address = 0x45
```

Reply:
Done!